assets/src/blocks/Spreadsheet/SpreadsheetEditorScript.js

```diff
@@ -15,17 +15,19 @@ import {
 
 import {SpreadsheetFrontend} from './SpreadsheetFrontend';
 
-const colors = [
+let colors = [
   {name: 'blue', color: '#c9e7fa'},
   {name: 'green', color: '#d0fac9'},
   {name: 'grey', color: '#ececec'},
 ];
 
 if (isNewIdentity) {
-  colors.push({
-    name: 'dark-green',
-    color: '#1f4912'
-  });
+  colors = [
+    {name: 'blue', color: '#167f82'},
+    {name: 'green', color: '#198700'},
+    {name: 'grey', color: '#45494c'},
+    {name: 'dark-green',color: '#1f4912'},
```

Comment:
Since this is a new color, we could rename it `gp-green` instead of `dark-green` to be kinda consistent with the Table block and the CSS variables

Reply:
So, the `gp-green` color code is actually what replaced the green, the `dark-green` is the new color added and matches the color code for the Table block

Reply:
well that shouldn't be the case, `green` should remain `green`, and for the Table block [the new color is named `gp-green`](https://github.com/greenpeace/planet4-master-theme/pull/1970/files#diff-01c9525afc2c87cfcb03124117c9d0ebb067029f6c955a607fa0fd4274aca556R614) so I think we should use the same here

Reply:
The colors should correspond between the blocks... So `#1f4912` should correspond with `green` and we can use `gp-green` for `#198700`... It's also what is described in the ticket with `#198700` being the new color